CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.10.1 (July 15, 2021)
 
+KNOWN ISSUES:
+
+* The change to enable streaming by default uncovered an incompatibility between streaming and WAN federation over mesh gateways causing traffic to fall back to attempting a direct WAN connection rather than transiting through the gateways. We currently suggest not upgrading beyond 1.10.0 if using WAN federation over mesh gateways, and are working to address this issue in a future patch release.
```

Comment:
Maybe we should recommend setting `use_streaming_backend=false` ([ref](https://www.consul.io/docs/agent/options#use_streaming_backend)) instead of staying on 1.10.0 ?  That would allow them to pickup other CVE and bug fixes from 1.10.1.

Reply:
Good call, yea that would be preferable.